src/keywords.YAML-tmLanguage

```diff
@@ -8,7 +8,7 @@ repository:
                 match: \b(in|is|as|assert)\b
                 name: keyword.operator.kotlin
             -
-                match: \b(const)\b
+                match: \b(const|lateinit)\b
```

Comment:
👍

Reply:
Oops, I just figured out `const` and `lateinit` both can also be used as variables, i.e. are soft keywords 😛 should've been with their friends!